R/get_chol.R

```diff
@@ -0,0 +1,58 @@
+#' Obtain Cholesky for new data and theta.
+#'
+#' @description Use a fitted model to obtain (weighted) Cholesky factors
+#' with new data, weights and theta.
+#'
+#' @param fit (`mmrm_fit`)\cr produced by [mmrm()].
+#' @param data (`data.frame`)\cr input data frame.
+#' @param weights (`numeric`)\cr vector of weights.
+#' @param theta (`numeric`)\cr new theta estimates.
+#' @param complete_case (`flag`)\cr whether to use complete input `data`.
```

Comment:
because we only need visit and subject id to obtain the subject Cholesky factors, so whether there is missing values in Y does not matter now. This option is added to ensure the consistency (with no default value, it give the same output as fit$tmb_object$report)